src/app/components/collection/collection.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="collection">
+    @if (isLoading) {
+    <div class="collection--loading">
+        <mat-spinner></mat-spinner>
+    </div>
+    }
+
+    <div class="collection__header">
+        <div class="collection__header__title">
+            {{'Collections' | translate}}
+        </div>
+        <div class="collection__header__filter">
+            @if (isAuthentActivated && authentMode === 'iam') {
+            <div class="collection__header__filter--public">
+                <mat-slide-toggle (change)="filterCollections()" [(ngModel)]="isPublic">{{'Is Public' |
+                    translate}}</mat-slide-toggle>
+            </div>
+            @if(organisations().length > 0){
+            <div class="collection__header__filter--org">
+                <mat-select [multiple]="true" [placeholder]="'Organisations' | translate"
+                    (selectionChange)="filterCollections()" [(value)]="organisationsNames">
+                    @for (org of organisations(); track $index) {
+                    <mat-option [value]="org">{{org}}</mat-option>
+                    }
+                </mat-select>
+            </div>
+            }
+
+            }
+            <div class="collection__header__filter--search">
+                <input matInput [(ngModel)]="searchValue" (ngModelChange)="search()"
+                    [placeholder]="'Filter collection on name' | translate" />
+            </div>
+        </div>
+    </div>
+    <div class="collection__content">
+        <table mat-table [dataSource]="collectionDataSource" (matSortChange)="sortData($event)" matSort
+            class="collection__content__table">
+
+            <ng-container matColumnDef="collection_name">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header> {{'Identifier' | translate}} </th>
+                <td mat-cell *matCellDef="let row">
+                    <a routerLink="/collection/edit/{{row.collection_name}}" class="detail-link"
+                        routerLinkActive="active">{{row.collection_name}}</a>
+                </td>
+            </ng-container>
+
+            <ng-container matColumnDef="display_name">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header> {{'Name' | translate}} </th>
+                <td mat-cell *matCellDef="let row">
+                    {{row.params.display_names.collection }}
+                </td>
+            </ng-container>
+
+            <ng-container matColumnDef="owner">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header> {{'Owner' | translate}} </th>
+                <td mat-cell *matCellDef="let row">
+                    {{row.params.organisations.owner}}
+                </td>
+            </ng-container>
+
+            <ng-container matColumnDef="shared_with">
```

Comment:
I think shared with should be displayed only when authentMode === 'iam'

Reply:
Already the case see here :
https://github.com/gisaia/ARLAS-wui-hub/blob/feat/collectionView/src/app/components/collection/collection.component.ts#L94